python/src/trezorlib/ui.py

```diff
@@ -54,6 +55,12 @@
 WIPE_CODE_NEW = PinMatrixRequestType.WipeCodeFirst
 WIPE_CODE_CONFIRM = PinMatrixRequestType.WipeCodeSecond
 
+# Workaround for limitation of Git Bash
+# getpass function does not work correctly on Windows when not using a real terminal
+# (the hidden input is not allowed and it also freezes the script completely)
+# Details: https://bugs.python.org/issue44762
+CAN_HANDLE_HIDDEN_INPUT = sys.stdin.isatty()
```

Comment:
`sys.stdin and sys.stdin.isatty()` (from the linked issue - `sys.stdin` can also be none, in which case we probably don't handle the hidden input either